client/i18n/locales/english/translations.json

```
@@ -383,6 +384,10 @@
     "percent-complete": "{{percent}}% complete",
     "project-complete": "Completed {{completedChallengesInBlock}} of {{totalChallengesInBlock}} certification projects",
     "tried-rsa": "If you've already tried the <0>Read-Search-Ask</0> method, then you can ask for help on the freeCodeCamp forum.",
+    "read-search-ask-checkbox": "I have tried the Read-Search-Ask method",
+    "similar-questions-checkbox": "I searched for similar questions that have already been answered on the forum",
+    "minimum-characters": "Add a minimum of {{characters}} more characters",
+    "characters-left": "{{characters}} characters left",
     "rsa": "Read, search, ask",
     "rsa-forum": "<strong>Before making a new post</strong> please see if your question has <0>already been answered on the forum</0>.",
```

```suggestion
    "rsa-forum": "<strong>Before making a new post</strong> please <0>check if your question has already been answered on the forum</0>.",
```

I would extend the link text so that it makes more sense to a screen reader user listening to just the link text.